admin/admin-service/pom.xml

```diff
@@ -7,7 +7,7 @@
 	<parent>
 		<groupId>io.mosip.admin</groupId>
 		<artifactId>admin-parent</artifactId>
-		<version>1.1.5.3-SNAPSHOT</version>
+		<version>1.1.5.4-SNAPSHOT</version>
```

Comment:
I see this version is changed, but the PR is still raised to 1.1.5.3

Reply:
taken care

Reply:
Resolved